ibis/sql/postgres/tests/conftest.py

```diff
@@ -19,9 +19,6 @@
 
 import ibis
 
-pytest.importorskip('sqlalchemy')
-pytest.importorskip('psycopg2')
```

Comment:
otherwise the w/o psycopg2 installed pytest completely fails to even collect. I don't think you are supposed to explicity skip in a conftest like this.